tests/serialization/test_serde.py

```diff
@@ -264,14 +264,33 @@ def test_backwards_compat(self):
             "__var": {
                 "__var": {
                     "uri": uri,
-                    "extra": None,
+                    "extra": {"__var": {"hi": "bye"}, "__type": "dict"},
                 },
                 "__type": "dict",
             },
         }
         dataset = deserialize(data)
         assert dataset.uri == uri
 
+    def test_backwards_compat_airflow_252(self):
+        """
+        Ensure that the Xcom value encoded by Airflow 2.5.2 is getting decoded by Airflow 2.6.
```

Comment:
I think you need to update this comment, since we have the same format since 2.4.3, you can just say < 2.6, wdyt?

Reply:
@hussein-awala Updated the comment, can you please check now? :)